src/nvidia-cuda/devcontainer-feature.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": "nvidia-cuda",
+  "name": "NVIDIA CUDA",
+  "description": "Installs NVIDIA CUDA libraries.",
+  "version": "1.0.0",
+  "options": {
+    "install_cuda": {
```

Comment:
What's the scenario where you wouldn't want to install the base CUDA libraries? Do we need to make this an option or should it just always happen?

Reply:
I can't think of a scenario, as this is the point of this feature 😄 I'll remove the option